mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'mtp_common/user_admin/sign-up-success.html' %}
+{% load i18n %}
+{% load mtp_common %}
+
+{% block whats_next_message %}
+  <p>
+    {% blocktrans trimmed with approver=approver|wrapwithtag:'strong' %}
+      Once {{ approver }} has approved your request, you’ll get an email with your sign in details.
```

Comment:
You can hardcode approver, as it's always going to be FIU

Reply:
🤔 It's true actual account request is always approved by FIU but in Justin prototype:
1. First request confirmation message says: ["Once **a member of the Financial Investigations Unit** has approved [...]"](https://pm-intelligence-tool.herokuapp.com/request_access_confirmation)
2. Subsequent request confirmation message says: ["Once **Family Services** has approved [...]"](https://pm-intelligence-tool.herokuapp.com/request_access_already_confirmation)

I'll double-check with Justin 🤷 

PS: This is also why when we submit a ticket to ZenDesk we set the approver to "Family Services".

Reply:
FYI: I was following the prototype, but turned out we actually don't want to show a different message so [I simplified the code](https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/531/commits/391efe0259e2c1d7539652fc0c47196b4d098e9d) and can resolve this and the other related comment.